CDTDatastoreTests/CDTSessionCookieInterceptorTests.m

```diff
@@ -137,6 +147,28 @@ - (void)testCookieInterceptorHandles401
 }
 
 /**
+ * Test cookie flow, where cookie credential expires
+ * - GET a resource on the cloudant server
+ * - Cookie jar empty, so get session cookie
+ * - Verify that credentials expired with 403 HTTP response and cookie should renew
+ */
+- (void)test403CredentialsExpiredRenewal
+{
+    [self basic403Test:@"credentials_expired" withReason:@"session expired" expectedRequests:5];
```

Comment:
I think this should actually be 6. See summary table in my other comment. I think the reason it asserts correctly is because the additional stub in setup isn't counted by the helper - another reason to remove that setup stub.